### PR TITLE
To organize commands better and for every command to have their own f…

### LINKED ISSUE #29: update commands to work with PREFIX
To keep things more organized we need to every command to have their own files and remove all the logic from the `index.js`. Also create a firestore configuration file to access the required variables in different files (needed for save, and flame)

`!flame help` to `!flame-help`
`!flame -r` to `!flame-random`
`flame -s` to `!flame-save`

### PR DESCRIPTION
**Pull request by jairoMolina9**
Removed `help.js` from root folder, and created the following files inside **commands** folder:
- flame-help.js
- flame-random.js
- flame-save.js
- flame.js

Added a firestore configuration file that exports `admin`, in way to access it from different files.

Modified `index.js`, removed unnecessary imports and cleaned command logic.

closes #29

### CODE REVIEW

File: commands/flame-save.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+module.exports = {
+  name: "flame-save",
+  description: "Saves flame to firestore",
+  execute(message, args) {
+
+    let user_id = ""; //outside bc docRef is const
+
+    try {
+      user_id = message.mentions.users.first().id;
```
Comment (edwincarbajal) [inline commands/flame-save.js:11]: If no user is provided, will this still work?

File: commands/flame.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+module.exports = {
+  name: "flame",
+  description: "Sends custom flame to user",
+  execute(message, args) {
+
+    //outside bc it is used in code below line 23
+    let user_id = "";
+    let mention = "";
+    try {
+      user_id = message.mentions.users.first().id;
```
Comment (edwincarbajal) [inline commands/flame.js:14]: Check if a user is provided and throw an error

File: commands/flame.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      })
+      .then(function (doc) {
+        if (doc.exists) {
+          let data = doc.data(); //array of custom flames
+          let index = Math.floor(Math.random() * data["flames"].length);
+
+          getRandom().then(function (giphy) {
+            message.channel
+              .send(
+                "🔥🔥 " + mention + ", " + data["flames"][index] + " 🔥🔥",
```
Comment (edwincarbajal) [inline commands/flame.js:36]: Let's continue using template literals

File: commands/flame.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                }
+              )
+              .then((sentMessage) => sentMessage.react("🔥"));
+          });
+        } else {
+          //if user has no custom flames
+          getRandom().then(function (giphy) {
+            message.channel
+              .send(
+                "🔥🔥 " + mention + ", " + insulter.default() + " 🔥🔥",
```
Comment (edwincarbajal) [inline commands/flame.js:48]: Let's continue using template literals

File: commands/flame-random.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    //outside bc it is used in code below line 23
+    let user_id = "";
+    let mention = "";
+    try {
+      user_id = message.mentions.users.first().id;
+      mention = "<@".concat(user_id).concat(">");
+    } catch (error) {
+      message.reply("mention an user!");
+      return;
```
[the remark this answers is not in the document]
Comment (jairoMolina9) [reply commands/flame-random.js:19]: yes to stop program

File: commands/flame-save.js (hunk 1 above)
Comment (jairoMolina9) [reply commands/flame-save.js:11]: yes

File: commands/flame-save.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      if(args.length == 1) throw ("noflame");
+
+    } catch (error) {
+      if(error === "noflame")
+        message.reply("write a flame!");
+      else
+        message.reply("mention an user!");
+
+      return;
```
[the remark this answers is not in the document]
Comment (jairoMolina9) [reply commands/flame-save.js:20]: yea to stop program?

File: commands/flame-save.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    let flameArray = [args[1]];
+
+    for(var i = 2; i < args.length; i++) {
+       flameArray[i-1] = args[i];
+    }
```
[the remark this answers is not in the document]
Comment (jairoMolina9) [reply commands/flame-save.js:27]: flameArray is taking the custom flame given by the user.

**Pull request merged**